lib/memo_wise.rb

```diff
@@ -34,25 +34,29 @@ def memo_wise(method_name)
         private not_memoized_name
 
         if instance_method(method_name).arity.zero?
-          method_name_and_args = "#{method_name}()"
-          key = method_name.inspect
+          module_eval <<-END_OF_METHOD, __FILE__, __LINE__ + 1
+            def #{method_name}
+              key = #{method_name.inspect}
```

Comment:
I know I was the one who originally led you to change this to `inspect`, but since we're elsewhere using `:#{method_name}` and this is in the middle of a bigger string interpolation now I'd vote we just change this back for consistency (or change the other instances of `:#{method_name}` to `#{method_name.inspect}`).